src/test/java/org/jabref/logic/l10n/LocalizationParser.java

```diff
@@ -259,6 +269,39 @@ public boolean containsKey(String key) {
             return result;
         }
 
+        public static List<String> getLocalizationParameter(String content, LocalizationBundleForTest type) {
+            List<String> result = new LinkedList<>();
+
+            Matcher matcher;
+            if (type == LocalizationBundleForTest.LANG) {
+                matcher = LOCALIZATION_START_PATTERN.matcher(content);
+            } else {
+                matcher = LOCALIZATION_MENU_START_PATTERN.matcher(content);
+            }
+            while (matcher.find()) {
+                // find contents between the brackets, covering multi-line strings as well
+                int index = matcher.end();
+                int brackets = 1;
+                StringBuilder buffer = new StringBuilder();
+                while (brackets != 0) {
```

Comment:
Thanks for the hints, but this is old code which I didnt write and don't wanna change if not really necessary.